assets/Product.wxs

```diff
@@ -128,8 +128,13 @@
             <Component Id="ProductVersionFolder" Guid="{e1a7f05e-0cd6-4227-80a8-e4fb311f045c}">
               <CreateFolder/>
             </Component>
-            <!-- register ourselves in application registry so can be started using just Win+R `pwsh.exe` -->
             <Component Id="RegistryEntries" Guid="{402e52f7-baf8-489d-af21-f756a6ca3530}">
+                <!-- create key for easy detection of a particular version of a powershell core package 
+                     The upgrade code is used in the key because it will change when we allow SxS       -->
+                <RegistryKey Root="HKLM" Key="Software\Microsoft\PowerShell Core\InstalledVersions\$(var.UpgradeCode)" ForceCreateOnInstall="yes" ForceDeleteOnUninstall="yes">
```

Comment:
We are using `Software\Policies\Microsoft\PowerShellCore` in GP settings, so maybe use 'PowerShellCore' here too.